core/src/main/java/org/apache/accumulo/core/metrics/MetricsProducer.java

```diff
@@ -52,6 +53,28 @@
  * <td></td>
  * </tr>
  * <tr>
+ * <td>N/A</td>
+ * <td>N/A</td>
+ * <td>{@value #METRICS_COMPACTOR_ENTRIES_READ}</td>
+ * <td>FunctionCounter</td>
+ * <td>Number of entries read by all compactions that have run on a compactor or tserver</td>
```

Comment:
The metric will be emitted per process. The user can select whether to look at the metrics at the individual process level (using one of the tags) or aggregate them and get a global view. Suggested re-working for the description below, same applies to entries written.

```suggestion
 * <td>Number of entries read by all threads performing compactions</td>
```